docs/managing-dragonfly/operator/prometheus-guide.md

```diff
@@ -0,0 +1,106 @@
+---
```

Comment:
We already have the prometheus guide right? https://www.dragonflydb.io/docs/managing-dragonfly/operator/prometheus-guide

Reply:
yep, it was added because the prom-guide PR was not merged